prediction_market_agent/data_models/market_data_models.py

```diff
@@ -10,6 +12,15 @@
 )
 
 
+class Condition(BaseModel):
+    id: HexAddress
+    outcomeSlotCount: int
+
+    @property
+    def index_sets(self) -> t.List[int]:
+        return [i + 1 for i in range(self.outcomeSlotCount)]
```

Comment:
Their indexes start at 1?

Reply:
@evangriffiths could you please clarify where did you learn that indexes start at 1? Couldn't find anything in the contract logic (https://github.com/gnosis/conditional-tokens-contracts/blob/master/contracts/ConditionalTokens.sol#L228)

Reply:
For future reference, I lifted it from the Olas trader here: https://github.com/valory-xyz/trader/blob/90a8f2f5a25f10d7c791f4223006cb0592d9cf77/packages/valory/skills/decision_maker_abci/redeem_info.py#L51

Reply:
Yes, the logic makes sense, see tx  -> https://gnosisscan.io/tx/0x1f9c94203e51ad67446853deae3ecf4ad7f5ea39bc5ebcd3d65afc8e9a91cb7f#eventlog